alkemio.yml

```diff
@@ -312,6 +312,70 @@ storage:
     # Max file size of files that will be uploaded to IPFS (in bytes).
     maxFileSize: 1048576
 
+notifications:
+  rabbitmq:
+    # Connection in the form of 'amqp://[user]:[password]@[host]:[port]?heartbeat=30'
+    connection:
+      # RabbitMQ host
+      host: ${RABBITMQ_HOST}:localhost
+
+      # RabbitMQ AMQP port. Used by AMQP 0-9-1 and 1.0 clients without and with TLS
+      port: ${RABBITMQ_PORT}:5672
+
+      # RabbitMQ user
+      user: ${RABBITMQ_USER}:user
+
+      # RabbitMQ password
+      password: ${RABBITMQ_PASSWORD}:bitnami
+
+    # PUBSUB configuration. Alkemio uses AMQP for messaging queue protocol - https://www.rabbitmq.com/tutorials/amqp-concepts.html.
+    pub_sub:
+      # RabbitMQ exchange. The RabbitMQ messaging model uses producer-queue-consumer.
+      # The core idea in the messaging model in RabbitMQ is that the producer never sends any messages directly to a queue.
+      # Actually, quite often the producer doesn't even know if a message will be delivered to any queue at all.
+      # Instead, the producer can only send messages to an exchange. An exchange is a very simple thing.
+      # On one side it receives messages from producers and the other side it pushes them to queues.
+      # The exchange must know exactly what to do with a message it receives. Should it be appended to a particular queue?
+      # Should it be appended to many queues? Or should it get discarded. The rules for that are defined by the exchange type.
+      exchange:
+        # RabbitMQ exchange name
+        name: ${RABBITMQ_EXCHANGE_NAME}:alkemio-graphql-subscriptions
```

Comment:
We will be using rabbitmq for a lot more I think than just graphql subscriptions - right?

Reply:
This is a default, can be changed with env variable. We can change later.